### ISSUE TITLE
[v5.1.0] Class '\DocuSign\eSign\Model\DateTime' not found

### ISSUE THREAD

**Issue by starvsion** [docusign/docusign-php-client#97]
Hi, Seems that the SDK has errors right now,  something to do with DateTime ... 

trace: 

```
#0 vendor/docusign/esign-client/src/ObjectSerializer.php(236): DocuSign\eSign\ObjectSerializer::deserialize(Object(stdClass), '\\DocuSign\\eSign...', NULL)
#1 vendor/docusign/esign-client/src/ObjectSerializer.php(291): DocuSign\eSign\ObjectSerializer::deserialize(Array, '\\DocuSign\\eSign...', NULL)
#2 vendor/docusign/esign-client/src/ObjectSerializer.php(291): DocuSign\eSign\ObjectSerializer::deserialize(Object(stdClass), '\\DocuSign\\eSign...', NULL)
#3 vendor/docusign/esign-client/src/ObjectSerializer.php(236): DocuSign\eSign\ObjectSerializer::deserialize(Object(stdClass), '\\DocuSign\\eSign...', NULL)
#4 vendor/docusign/esign-client/src/ObjectSerializer.php(291): DocuSign\eSign\ObjectSerializer::deserialize(Array, '\\DocuSign\\eSign...', NULL)
#5 vendor/docusign/esign-client/src/Api/TemplatesApi.php(5743): DocuSign\eSign\ObjectSerializer::deserialize(Object(stdClass), '\\DocuSign\\eSign...', Array)
#6 vendor/docusign/esign-client/src/Api/TemplatesApi.php(5651): DocuSign\eSign\Api\TemplatesApi->listRecipientsWithHttpInfo('5c5ac154-2a7e-4...', 'cfc2e3a5-0032-4...', Object(DocuSign\eSign\Api\TemplatesApi\ListRecipientsOptions))
#7 Docusign/EnvelopeGenerator.php(162): DocuSign\eSign\Api\TemplatesApi->listRecipients('5c5ac154-2a7e-4...', 'cfc2e3a5-0032-4...', Object(DocuSign\eSign\Api\TemplatesApi\ListRecipientsOptions))
```

**Comment by dbbrahmbhatt** [comment]
Hi @starvsion . Thanks for reporting this issue. It has been fixed in our latest release v5.2.0. Can you please verify?

**Comment by LarryKlugerDS** [comment]
No update since Jun 5....closing. Please respond if this is still an issue. Thank you.
